### ISSUE TITLE
When MainForm are hidden,MainForm 's MaterialDrawer can't be hidden

### ISSUE THREAD

**Issue by cxw3292008** [leocb/MaterialSkin#27]
Hi ,

When MainForm are hidden,MainForm 's MaterialDrawer can't be hidden
this.Visible = false;

![image](https://user-images.githubusercontent.com/8295531/68023313-cf767980-fce1-11e9-9f08-16ff317dfba2.png)

![image](https://user-images.githubusercontent.com/8295531/68023330-da310e80-fce1-11e9-93b3-182428b5aaec.png)

**Comment by leocb** [comment]
Hmm yes, that's indeed a bug, I did not think of this use case.
This makes sense as I use a second form for the drawer. And now, thinking better about it, I can think of some other moments where this can also happen. Will fix for the next release.
If you're not using the pre-compiled dll and want to fix it yourself, on the `MaterialForm.cs` file, on the `AddDrawerOverlayForm` method, after the `// Resize and move events` comment, add this:
```C#
            VisibleChanged += (sender, e) =>
            {
                drawerForm.Visible = Visible;
                drawerOverlay.Visible = Visible;
            };
```

**Comment by leocb** [comment]
Fixed on #28 Though I do not recommend you use the latest master for now, some breaking bugs still occur and I'm still hunting them down.

**Comment by cxw3292008** [comment]
@leocb  Thank you very much.O(∩_∩)O
